early-attack-detector-py/src/utils.py

```diff
@@ -18,7 +22,7 @@ def calc_contract_address(w3, address, nonce) -> str:
     """
 
     address_bytes = bytes.fromhex(address[2:].lower())
-    return Web3.toChecksumAddress(Web3.keccak(rlp.encode([address_bytes, nonce]))[-20:])
+    return Web3.to_checksum_address(Web3.keccak(rlp.encode([address_bytes, nonce]))[-20:])
```

Comment:
Do these changes work fine for you? I think we need the camel case for web3 v5 (unless there's something wrong with my local env)

Reply:
Yeah, they did. And as a matter of fact, it was erroring out for me that I needed to change it. 🤔

Reply:
In trying to address the errors with `scikit-learn`, I encountered errors with `to_checksum_address`. So now changed it back to `toChecksumAddress`.